bin/content-style

```diff
@@ -25,10 +26,21 @@ violations = files.flat_map do |file|
   file_content = File.read(file)
   file_path = File.expand_path(file)
   next if excluded_file_paths.include?(file_path)
+  next if file_content.empty?
+  hotcop_identifiers = ContentStyle::Parser.get_erb_locations(file_content)
   linter.lint_file(ContentStyle::Parser.parse(file_content)).map do |v|
-    m = v[:message]
+    hotcop_identifiers.reverse.each do |id|
+      @erb_location = id[:erb_location] if v[:line] > id[:line]
+      next if v[:line] < id[:line]
+    end
```

Comment:
This is logic that any consumer of `ContentStyle::Parser.get_erb_locations` will have to implement, so it's a little challenging to use. 

My recommendation for `get_erb_locations`: return an array whose size is equal to the number of lines in the file and each entry is the associated ERB location. That way you can discard this logic and simply do `@erb_location = hotcop_identifiers[v[:line]]`.

Another option is to make `ContentStyle::Parser.parse` return an object that encapsulates all of these nitty gritty details. Your code here may look like this then:
```ruby
violations = files.flat_map do |file|
  file_path = File.expand_path(file)
  next if excluded_file_paths.include?(file_path)

  # file_content = File.read(file)
  # next if file_content.empty?
  # ^-- becomes
  parsed_content = ContentStyle::Parser.parse(file)

  # hotcop_identifiers = ContentStyle::Parser.get_erb_locations(file_content)
  # ^-- becomes (see first line of loop below)

  # Note the use of parsed_content.xml_tree, a method on the result of the parse method above
  linter.lint_file(parsed_content.xml_tree).map do |v|
    @erb_location = parsed_content.source_erb_file(v[:line])

    erb_location_path = Pathname.new(File.expand_path(@erb_location))
    e = erb_location_path.relative_path_from(Dir.pwd)
    f = file_path
    t = v[:text]
    l = v[:line]
    m = v[:message]
    "#{f},#{l},#{t},#{m},#{e}"
  end
end
```
You can do the same for `linter.lint_file`. Instead of returning a hash, it could return a small `LintResult` object that has a `to_s` method that does all of that string interpolation you're doing here.